src/App.js

```diff
@@ -63,13 +63,14 @@ function allowTabs() {
 
 export default function App() {
   // TODO change back to not dev
-  const BACKEND_URL = `https://cjsback.herokuapp.com/`
+  const BACKEND_URL = `https://cjsbackdev.herokuapp.com/`
```

Comment:
should this be switched back to the prod server?

Reply:
+1